Description: Fix licence
 The software is released under GPLv3+ but the help message displays GPLv2+.
 This patch fix the help message for consistency purpose.
Author: Franck Joncourt <franck.joncourt@gmail.com>
Applied-Upstream:1.4, commit:https://github.com/jp-bennett/fwknop-gui/tree/a3991f7c75de5d3358cb56f552f833d834573fb0

--- a/help.html
+++ b/help.html
@@ -11,7 +11,7 @@
   Once the data is filled in, select "Save Config" from the menu. Then, to send a SPA knock, select the
    Saved config you want to use, and click the "Send Knock" button at the bottom of the list.
  </p>
- <p>This app is open source software, Licensed GPLv2+. The source can be found  <a href="https://github.com/jp-bennett/fwknop-gui">on github</a>. Please report any bugs there.</p>
+ <p>This app is open source software, Licensed GPLv3+. The source can be found  <a href="https://github.com/jp-bennett/fwknop-gui">on github</a>. Please report any bugs there.</p>
  <p>Credits: Based on fwknop and libfko written by <a href="https://www.cipherdyne.org/fwknop/"> Michael Rash</a></p>
  <p>This app was written by <a href="http://incomsystems.biz/"> Jonathan Bennett</a></p>
  <p> The fwknop-gui icon is derived from on a photo by <a href="https://www.flickr.com/photos/anemoneprojectors/5330583056/">Peter O'Connor</a></p>
